packages/xo-server/src/xo-mixins/metadata-backups.js

```
@@ -121,7 +144,24 @@ export default class metadataBackup {
           return Promise.all([
             handler.outputFile(fileName, data),
             handler.outputFile(metaDataFileName, metadata),
-          ])
+          ]).then(
+            result => {
+              logger.notice(`Backuping XO metadata is a success. (${job.id})`, {
+                event: 'task.end',
+                status: 'success',
+                taskId,
+              })
+              return result
+            },
+            error => {
```

Is it intended to swallow the error?